test/functional/p2p_feefilter.py

```diff
@@ -57,6 +57,9 @@ def set_test_params(self):
         # rounding down 3 places, leading to stranded transactions.
         # See issue #16499
         self.extra_args = [["-minrelaytxfee=0.00000100", "-mintxfee=0.00000100"]] * self.num_nodes
+        # whitelist all peers to speed up tx relay / mempool sync
+        for args in self.extra_args:
+            args.append("-whitelist=noban@127.0.0.1")
```

Comment:
could just inline this arg with the others and put the comment above line 59

Reply:
Done.